applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -779,6 +867,30 @@ impl AppStateInner {
         Ok(())
     }
 
+    pub async fn refresh_burnt_proofs_state(&mut self) -> Result<(), UiError> {
+        let db_burnt_proofs = self.wallet.db.get_burnt_proofs()?;
+        let mut ui_proofs: Vec<UiBurntProof> = vec![];
+
+        for proof in db_burnt_proofs {
+            ui_proofs.push(UiBurntProof {
+                id: proof.0,
+                reciprocal_claim_public_key: proof.1,
+                payload: proof.2,
+                burned_at: proof.3,
+            });
+        }
+
+        ui_proofs.sort_by(|a, b| {
+            a.burned_at
+                .partial_cmp(&b.burned_at)
```

Comment:
Why not just use cmp so that we can avoid the expect?
```suggestion
                .cmp(&b.burned_at)
```